tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -371,23 +376,21 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'Available transformations: "LowLatency2", "MakeStateful"',
                               default="")
     common_group.add_argument('--disable_fusing',
-                              help='Turn off fusing of linear operations to Convolution',
+                              help='[DEPRECATED] Turn off fusing of linear operations to Convolution',
                               action=DeprecatedStoreTrue)
     common_group.add_argument('--disable_resnet_optimization',
-                              help='Turn off resnet optimization',
-                              action='store_true')
+                              help='[DEPRECATED] Turn off resnet optimization',
+                              action=DeprecatedStoreTrue, default=False)
     common_group.add_argument('--finegrain_fusing',
-                              help='Regex for layers/operations that won\'t be fused. ' +
-                                   'Example: --finegrain_fusing Convolution1,.*Scale.*')
+                              help='[DEPRECATED] Regex for layers/operations that won\'t be fused. ' +
+                                   'Example: --finegrain_fusing Convolution1,.*Scale.*',
+                              action=DeprecatedOptionCommon)
     common_group.add_argument('--disable_gfusing',
-                              help='Turn off fusing of grouped convolutions',
-                              action=DeprecatedStoreTrue)
+                              help=argparse.SUPPRESS,
+                              action=IgnoredAction)
     common_group.add_argument('--enable_concat_optimization',
-                              help='Turn on Concat optimization.',
-                              action='store_true')
-    common_group.add_argument('--move_to_preprocess',
```

Comment:
it was deprecated, can we just delete it?

Reply:
According to the process let us deprecate this for 2022.1 release first. And after this we will remove it at all